client/python/cryoet_data_portal/src/cryoet_data_portal/templates/Model.jinja2

```diff
@@ -43,14 +43,14 @@ class {{ model.name }}(Model):
     find.__func__.__doc__ = Model.find.__func__.__doc__ + find.__func__.__doc__
 
     @classmethod
-    def get_by_id(cls, **kwargs):
+    def get_by_id(cls, *args, **kwargs):
         """
         Examples:
             Get an {{ model.name }} by ID:
             >>> {{ model.model_name_underscores }} = {{ model.name }}.get_by_id(client, 1)
```

Comment:
Also not a blocker, but we should find a way to insert class-specific IDs here. The `id=1` will return `None` for almost every important model in the client, so these are not necessarily intuitive examples. I will create a ticket.

Reply:
Captured here: https://github.com/chanzuckerberg/cryoet-data-portal/issues/1381